Content.Shared/Cuffs/SharedCuffableSystem.cs

```diff
@@ -380,33 +380,25 @@ private void OnCuffVirtualItemDeleted(EntityUid uid, HandcuffComponent component
         /// <summary>
         ///     Check the current amount of hands the owner has, and if there's less hands than active cuffs we remove some cuffs.
         /// </summary>
-        private void OnHandCountChanged(HandCountChangedEvent message)
+        private void OnHandCountChanged(Entity<CuffableComponent> ent, ref HandCountChangedEvent message)
         {
-            var owner = message.Sender;
-
-            if (!TryComp(owner, out CuffableComponent? cuffable) ||
-                !cuffable.Initialized)
-            {
-                return;
-            }
-
             var dirty = false;
-            var handCount = CompOrNull<HandsComponent>(owner)?.Count ?? 0;
+            var handCount = CompOrNull<HandsComponent>(ent.Owner)?.Count ?? 0;
 
-            while (cuffable.CuffedHandCount > handCount && cuffable.CuffedHandCount > 0)
+            while (ent.Comp.CuffedHandCount > handCount && ent.Comp.CuffedHandCount > 0)
             {
                 dirty = true;
 
-                var container = cuffable.Container;
+                var container = ent.Comp.Container;
                 var entity = container.ContainedEntities[^1];
 
                 _container.Remove(entity, container);
-                _transform.SetWorldPosition(entity, _transform.GetWorldPosition(owner));
+                _transform.SetWorldPosition(entity, _transform.GetWorldPosition(ent.Owner));
```

Comment:
Can you update this to use PlaceNextTo instead?

Reply:
I also renamed some vars to be more readable.